src/controllers/ExperimentController.ts

```diff
@@ -28,7 +29,10 @@ export class ExperimentController {
      */
     insertImageVariableToFrame = async (imageFrameId: Id, variableId: string) => {
         const res = await this.#editorAPI;
-        return res.assignImageVariable(imageFrameId, variableId).then((result) => getEditorResponseData<null>(result));
+        const src: ImageUrlVariableSource = { variableId: variableId, sourceType: ImageSourceTypeEnum.urlVariable };
```

Comment:
That url (temporary) specific source was a suggestion by me - as for some reason - I thought it was the only use WRS did of `insertImageVariableToFrame` but it actually seems to not be used at all...

I'm not sure if @srares76 or @pkgacek  are using it in a current ticket but maybe we should revert back to `assignImageVariable` for now and refactor it when it's time for it > [EDT-758](https://support.chili-publish.com/browse/EDT-758). Unless you think otherwise Evan, i'd say it's the way to go to avoid any issue.

Reply:
what's about this comment now @Dvergar ?

Reply:
@brapoprod maybe you have some thoughts one that one? 
Basically the next ticket will fix this issue, but what will be the best option for now?

Reply:
UPD

Resolved for now by reverting back `assignImageVariable(imageFrameId, variableId)` method on the editor side to avoid any breaking changes atm